features/Spec.groovy

```diff
@@ -0,0 +1,17 @@
+def helpers = new ch.softozor.pipeline.Helpers()
+
+pipeline {
+  agent any
+  stages {
+    stage('Spec generation') {
+      environment {
+        SOFTOZOR_CREDENTIALS = credentials('softozor-credentials')
+      }
+      steps {
+        script {
+          helpers.generateSpecification("$WORKSPACE/features", 'backend', GIT_COMMIT)
```

Comment:
what is this `GIT_COMMIT` thing ?

Reply:
it's an environment variable defined on Jenkins; it contains the current commit number.